cfgov/unprocessed/css/molecules/social-media.less

```diff
@@ -36,62 +36,72 @@
 */
 
 .m-social-media {
-  display: inline-block;
-
-  // This could be wrapped in the `m-social-media__share` modifier,
-  // but it's ok on its own too.
-  &_heading {
-    .heading-5();
-    display: inline-block;
-    // 0.25em subtracted to compensate for inline spacing
-    padding-right: unit(@grid_gutter-width / @font-size - 0.25em, em);
-    margin-bottom: 0;
-    vertical-align: middle;
-  }
+  display: inline-flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  gap: unit(10px / @base-font-size-px, rem);
 
   .m-list {
```

Comment:
Let's update this to `.m-social-media_icons`, so that we're not directly customizing the `m-list` pattern. We're still customizing it by adding the `.m-social-media_icons` class, but at least the customizations are then in its own class. 

I think overall the usage of an unordered list in the social media component should be scrutinized, but that can happen another time.